src/components/donation/Steps/DonateMethods/Stripe/Incrementers.tsx

```diff
@@ -0,0 +1,31 @@
+import { useFormContext } from "react-hook-form";
+import { FormValues } from "./types";
+
+export default function Incrementers() {
+  return (
+    <div className="flex justify-center gap-3">
+      <Incrementer value={40} />
+      <Incrementer value={100} />
+      <Incrementer value={200} />
+    </div>
+  );
+}
+
+function Incrementer({ value }: { value: number }) {
+  const { setValue, trigger, watch } = useFormContext<FormValues>();
+  return (
+    <button
```

Comment:
```suggestion
    <button
       type="button"
```
so that doesn't go to next step (submits) - specify `type="submit"` if the it's the intention